lib/cc/cli/init.rb

```diff
@@ -30,7 +30,8 @@ def create_codeclimate_yaml
 
         config["exclude_paths"] = exclude_paths(AUTO_EXCLUDE_PATHS)
 
-        File.write(filesystem.path_for(CODECLIMATE_YAML), config.to_yaml)
+        filesystem.write(CODECLIMATE_YAML, config.to_yaml)
+        filesystem.correct_permissions(CODECLIMATE_YAML)
```

Comment:
Is it likely that we'll want to correct every file we write? I think probably. If so, we could move the `chown` into `FileSystem#write` itself, and remove `#correct_permissions` altogether.